code_of_conduct.MD

```diff
@@ -1,42 +1,5 @@
-# Code of Conduct
+# Code of conduct
```

Comment:
So, this file has essentially been moved to the website, but I left the COC here, as people expect to see codes of conduct at the base of the repository, which then links to the website.